tests/unit_tests/community/test_authentication.py

```diff
@@ -0,0 +1,343 @@
+#  Drakkar-Software OctoBot
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import pytest
+import mock
+import requests
+import time
+
+import octobot.community as community
+import octobot_commons.config_manager
+import octobot_commons.constants as commons_constants
+
+AUTH_URL = "https://oh.fake/auth"
```

Comment:
Could we use `os.getenv("COMMUNITY_SERVER_URL", "https://oh.fake/auth")` to test with a real server ? (same for www.todo.com below)